http4s/src/test/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorRoutesSpec.scala

```diff
@@ -69,6 +72,18 @@ class CollectorRoutesSpec extends Specification {
       response.as[String].unsafeRunSync() must beEqualTo("OK")
     }
 
+    "respond to the cors route with a preflight response" in {
+      val (_, routes) = createTestServices
+      def test(uri: Uri) = {
+        val request  = Request[IO](method = Method.OPTIONS, uri = uri)
+        val response = routes.run(request).unsafeRunSync()
```

Comment:
At some point we should use `cats.effect.testing.specs2.CatsIO` to remove these `.unsafeRunSync()` from the tests.

Reply:
Yep, that would be good. Would it be okay if I do that in the next PR ?

Reply:
Sure!